client/driver/executor/executor_linux.go

```diff
@@ -186,6 +259,26 @@ func (e *UniversalExecutor) removeChrootMounts() error {
 	return e.ctx.AllocDir.UnmountAll()
 }
 
+func (e *UniversalExecutor) getAllPids() ([]*nomadPid, error) {
```

Comment:
I would comment that we have a separate definition since when there is cgroup isolation we can do the pid scan much more efficiently